include/openPMD/IO/ADIOS/ADIOS1IOHandler.hpp

```diff
@@ -25,7 +25,6 @@
 #if openPMD_HAVE_ADIOS1
 #   include "openPMD/IO/AbstractIOHandlerImpl.hpp"
 #   include <adios.h>
```

Comment:
hm, `adios.h` also includes `adios_mpi.h`... argh

Reply:
I think you need `adios.h` and `mpi.h` in `ParallelADIOS1...hpp` only.

In the serial header, use `adios_read.h` and in the serial impl. `mpidummy.h` as you do.